src/main/java/Duke.java

```diff
@@ -1,10 +1,150 @@
+import exceptions.InvalidSyntaxException;
+import exceptions.UnrecognizedInputException;
+import java.util.List;
+import java.util.Scanner;
+import storage.TaskStorageManager;
+import task.DeadlineTask;
+import task.EventTask;
+import task.Task;
+import ui.Command;
+
 public class Duke {
+
+    private static final TaskStorageManager taskStorageManager = new TaskStorageManager();
+    private static final List<Task> userTasks = taskStorageManager.loadTasks();
```

Comment:
This is a good point; thanks for raising this! 

Since `taskStorageManager` and `userTasks` are not immutable objects, I do not think these variables should be stylized as constants. I'm not sure if this is explicitly stated in CS2113's guidelines, but an example of this would be [Google's Java Style Guide](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names).

Nonetheless, this was very helpful since I realized some of my constants in other areas were in fact mutable!